lib/ansible/template/__init__.py

```diff
@@ -810,16 +810,16 @@ def _now_datetime(self, utc=False, fmt=None):
 
         return now
 
-    def _query_lookup(self, name, *args, **kwargs):
+    def _query_lookup(self, ansible_lookup_name, *args, **kwargs):
```

Comment:
Use positional-only arguments instead:

```suggestion
    def _query_lookup(self, name, /, *args, **kwargs):
```